DishBookIOS/Classes/Coordinators/MainFlow/NewDishCoordinator.swift

```diff
@@ -6,23 +6,63 @@
 //
 
 import UIKit
+import Combine
 
 final class NewDishCoordinator: BaseRootCoordinator {
-        
+    
+    private var newDish = NewDish()
+    private var cancelableSet: Set<AnyCancellable> = []
```

Comment:
Maybe add this variable to BaseFlowCoordinator or something like this?

Reply:
cancelableSet?

Reply:
Yes

Reply:
Like in BaseViewModel and BaseViewController